client/packages/requisitions/package.json

```diff
@@ -9,7 +9,7 @@
     "husky": "^8.0.1",
     "lint-staged": "13.2.0",
     "prettier": "^2.3.2",
-    "typescript": "^4.3.5"
+    "typescript": "^5.0.2"
```

Comment:
can we move the common packages to client ? (tried with typescript, seems to work), I guess this can be done in a new issue, just saw multiple updates to the same package (and saw some discrepancy between how version is specified in some packages)

Reply:
nice, will do